docs/docs/index.mdx

```diff
@@ -7,6 +7,15 @@ title: Introduction to Rasa Action Servers
 A Rasa action server runs [custom actions](https://rasa.com/docs/rasa/next/custom-actions) for a Rasa Open Source
 conversational assistant.
 
+Rasa will call an endpoint you can specify, when a custom action is predicted. This endpoint should be a webserver that reacts to this call, runs the code and optionally returns information to modify the dialogue state.
+
+To specify your action server use the endpoints.yml:
+
+```yaml
```

Comment:
there is also a `title` tag for codeblocks. I think we should add that as it makes clear that this should go to the `endpoints.yml`